internal/server/data/migrations_test.go

```diff
@@ -0,0 +1,65 @@
+package data
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/infrahq/infra/internal/logging"
+	"github.com/infrahq/infra/internal/server/models"
+	"go.uber.org/zap/zaptest"
+	"gorm.io/gorm"
+	"gotest.tools/v3/assert"
+)
+
+func Test202204111503(t *testing.T) {
+	db := setupWithNoMigrations(t, func(db *gorm.DB) {
+		loadSQL(t, db, "202204111503")
```

Comment:
Any thoughts on where we can document the steps to create this file? Maybe as a comment in this file somewhere?